dashboard/modules/state/state_head.py

```diff
@@ -385,7 +385,12 @@ async def list_logs(self, req: aiohttp.web.Request) -> aiohttp.web.Response:
                 node_id, timeout, glob_filter=glob_filter
             )
         except DataSourceUnavailable as e:
-            return self._reply(success=False, error_message=str(e), result=None)
+            return self._reply(
+                success=False, 
+                error_message=str(e), 
+                result=None,
+                reason=str(e),
```

Comment:
nit, what if we use the `error_message` field as reason of `json_reponse` later?

Reply:
So basically something like this ?:
```python
    return aiohttp.web.json_response(
        {
            "result": success,
            "msg": message,
            "data": to_google_style(kwargs) if convert_google_style else kwargs,
        },
        dumps=functools.partial(json.dumps, cls=CustomEncoder),
        headers=headers,
        status=200 if success else 500,
        reason=None if success else message,
    )
```
without adding the additional parameter `reason`.

In this case, should behave the same, the question I guess would be: do we always expect the reason to be the same as the error message?

Reply:
> In this case, should behave the same, the question I guess would be: do we always expect the reason to be the same as the error message?

Good question - I don't have a strong opinion on this though. I think as long as it's interepred the same on the client side its fine.